app/views/admin/submissions/_collector_info.html.erb

```diff
@@ -9,8 +9,8 @@
     <div>
       <%= submission.name %>
     </div>
-    <div>
-      <%= submission.count_submissions_of_user %> submissions made
+    <div class="sidebar-link-2" >
+      <%= link_to "#{submission.count_submissions_of_user} submissions made", admin_submissions_path(user: submission.user_id) %>
```

Comment:
**question (blocking):** What will happen if an admin clicks on this link for an anonymous submission?

Reply:
anonymous submission also had a user_id, that is, it will select all submissions with this user_id, if the user_id is still not specified, then it will be redirected to an empty page with submissions
<img width="1498" alt="Screen Shot 2021-12-17 at 4 02 44 PM" src="https://user-images.githubusercontent.com/21379857/146548611-11628ec1-a32e-49e0-99f2-c809b83e6aad.png">

Reply:
Anonymous submissions shouldn't have a `user_id`, so I think that admins will always see a blank page if they click this link from an anonymous submission.

If 10 submissions were made with the same email address, then an admin would expect to see 10 submissions in the list when they click this link. However, if 1 of those 10 submissions is an anonymous submission, and they click this link from that anonymous submission, then they will see an empty list.

Reply:
I double-checked and this is a really great note, I will think about how to fix it

Reply:
@Gon4ar5 I wasn't able to see how you addressed this from the latest commits to this PR. Would you mind providing a little explanation, please?

Reply:
I added a new filter, and now it works like this: if a submission has a user, we display all submissions of this user, if not, then all submissions with the same user_email